app/assets/stylesheets/application.css.scss

```diff
@@ -48,6 +48,19 @@
   position: relative;
 }
 
+.admin-photo-container {
+  background-image: url('https://dancer.com/wp-content/uploads/2013/10/Cross-Training.jpg');
```

Comment:
Prefer double-quoted strings<br>URL `https://dancer.com/wp-content/uploads/2013/10/Cross-Training.jpg` should not contain protocol or domain